### DISCUSSION TITLE
Navigation of views with MvvmCross 7

### DISCUSSION THREAD

**Discussion by BloodLind** [MvvmCross/MvvmCross#4289] [Q&A]
I have WPF (.Net framework) applications. I am trying to implement a view which will be able to show certain views with its own ViewModels. 
I have been trying to figure out how to do this for a long time, but I found the same problems, but for Xamarin, the solution does not suit me. 
Can anyone explain how to make such a navigation. Yes, I saw an example in the Playground, but there is version 8, and I have a project on the previous one. Since a lot has already been done, I need help with the 7th version

**Comment by Cheesebaron** [comment]
> I am trying to implement a view which will be able to show certain views with its own ViewModels. 

This doesn't really explain what you are trying to do, please elaborate.

**Reply by BloodLind** [reply]
I have a page with tabs. Each tab has a view model under the hood. All these tabs are under the control of one view model with own view. 
![изображение](https://user-images.githubusercontent.com/69533604/137628894-424e7480-f24d-40c5-8fe9-ceec06d4a1d0.png)
In this screenshot, there is a menu on the left and a content view on the right. With the usual mvvm cross navigation, I do not get the desired result, since the content fills the entire window, and not a specific area for it on right.
![изображение](https://user-images.githubusercontent.com/69533604/137629051-11b199aa-e2d1-45d6-93f4-f6953de7242c.png)
In this screenshot from playground app, there it's work how I want. But when I try to implent this type of navigation I return to my problem, that my tabs fill all space in the entrie window.
I hope I described my problem correctly and clearly. Thanks for your help in advance.
